main.py

```diff
@@ -381,24 +381,21 @@ def enable_interface(self, interface_enable_id=None, dpid=None):
 
             try:
                 switch.interfaces[interface_number].enable()
-            except KeyError as exc:
-                error_list.append(f"Switch {dpid} Interface {exc}")
+            except KeyError:
+                msg = f"Switch {dpid} interface {interface_number} not found"
+                return jsonify(msg), 404
```

Comment:
In the future, like we have on `mef_eline` we can start raising `werkzeug.exceptions` but I minimized changes to facilitate the review.

Reply:
Yes, we should start raising those exceptions on every NApp

Reply:
I've mapped it here https://github.com/kytos-ng/topology/issues/52